datasets/1.0/simple-parquet/output/persons.jsonl

```diff
@@ -1,10 +1,10 @@
-{"name": "person0", "age": 0}
-{"name": "person1", "age": 1}
-{"name": "person2", "age": 2}
-{"name": "person3", "age": 3}
-{"name": "person4", "age": 4}
-{"name": "person5", "age": 5}
-{"name": "person6", "age": 6}
-{"name": "person7", "age": 7}
-{"name": "person8", "age": 8}
-{"name": "person9", "age": 9}
+{"persons/name": "person0", "persons/age": 0}
```

Comment:
I think it was feature to not have the RecordSet ID here - otherwise, it's repeated:

- The user is asking for the `persons` RecordSet
- => All fields will start with `persons`

Maybe, it's a consequence of my other comment where you set `name == @id` for all datasets. It's the case for Hugging Face datasets, but it may also not be the case per the specs.

So it could be good to keep at least one dataset with `name != @id` for testing purposes. What do you think?

Reply:
Sure! I updated all datasets because of the migration script, but good point to keep one different.